{"name": "Проверка валидности поля email", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'некорректный e-mail')]\")).text\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный e-mail')]\"}\n  (Session info: chrome=111.0.5563.65)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678768814622\\1678768814623.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678768814622\\1678768814623.html", "trace": "..\\venv\\lib\\site-packages\\selene\\core\\wait.py:102: in for_\n    return fn(self._entity)\n..\\venv\\lib\\site-packages\\selene\\core\\wait.py:56: in __call__\n    return self._fn(entity)\n..\\venv\\lib\\site-packages\\selene\\core\\query.py:47: in <lambda>\n    text: Query[Element, str] = Query('text', lambda element: element().text)\n..\\venv\\lib\\site-packages\\selene\\core\\entity.py:179: in __call__\n    return self._locator()\n..\\venv\\lib\\site-packages\\selene\\core\\locator.py:34: in __call__\n    return self._locate()\n..\\venv\\lib\\site-packages\\selene\\core\\entity.py:1529: in <lambda>\n    f'{self}.element({by})', lambda: self.driver.find_element(*by)\n..\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:976: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n..\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:321: in execute\n    self.error_handler.check_response(response)\n..\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py:242: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный e-mail')]\"}\nE     (Session info: chrome=111.0.5563.65)\n\nDuring handling of the above exception, another exception occurred:\ntest_login.py:77: in test_field_email\n    page.registration_field_email()\n..\\pages\\home.py:126: in registration_field_email\n    message_error = self.get_element_text(self.message_error_email_text_var, \"Текст ошибки поля  пароль не верный или текст отсутствует\")\n..\\pages\\base.py:16: in get_element_text\n    return element.get(query.text)\n..\\venv\\lib\\site-packages\\selene\\core\\entity.py:116: in get\n    return self.wait.for_(query)\n..\\venv\\lib\\site-packages\\selene\\core\\wait.py:125: in for_\n    raise self._hook_failure(failure)\nE   selene.core.exceptions.TimeoutException: Message: \nE   \nE   Timed out after 10s, while waiting for:\nE   browser.element(('xpath', \"//div[contains(text(),'некорректный e-mail')]\")).text\nE   \nE   Reason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный e-mail')]\"}\nE     (Session info: chrome=111.0.5563.65)\nE   \nE   Screenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678768814622\\1678768814623.png\nE   PageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678768814622\\1678768814623.html"}, "steps": [{"name": "Нажать Аккаунт", "status": "passed", "start": 1678768817417, "stop": 1678768817520}, {"name": "Нажать Зарегистрироваться", "status": "passed", "start": 1678768817520, "stop": 1678768817645}, {"name": "Валидация поля Email", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'некорректный e-mail')]\")).text\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный e-mail')]\"}\n  (Session info: chrome=111.0.5563.65)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678768814622\\1678768814623.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678768814622\\1678768814623.html\n\n", "trace": "  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\home.py\", line 126, in registration_field_email\n    message_error = self.get_element_text(self.message_error_email_text_var, \"Текст ошибки поля  пароль не верный или текст отсутствует\")\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\base.py\", line 16, in get_element_text\n    return element.get(query.text)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 116, in get\n    return self.wait.for_(query)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\wait.py\", line 125, in for_\n    raise self._hook_failure(failure)\n"}, "steps": [{"name": "Заполнение поля '' текстом ''mail''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000001B11621C640>"}, {"name": "text", "value": "'mail'"}, {"name": "fieldName", "value": "''"}], "start": 1678768817645, "stop": 1678768817724}, {"name": "Взять текст из элемента ''Текст ошибки поля  пароль не верный или текст отсутствует''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000001B11621E940>"}, {"name": "allureText", "value": "'Текст ошибки поля  пароль не верный или текст отсутствует'"}], "start": 1678768824782, "stop": 1678768824803}, {"name": "Заполнение поля '' текстом ''mail.mail''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000001B11621C640>"}, {"name": "text", "value": "'mail.mail'"}, {"name": "fieldName", "value": "''"}], "start": 1678768826815, "stop": 1678768826905}, {"name": "Взять текст из элемента ''Текст ошибки поля  пароль не верный или текст отсутствует''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000001B11621E940>"}, {"name": "allureText", "value": "'Текст ошибки поля  пароль не верный или текст отсутствует'"}], "start": 1678768833951, "stop": 1678768833968}, {"name": "Заполнение поля '' текстом ''mail@mail''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000001B11621C640>"}, {"name": "text", "value": "'mail@mail'"}, {"name": "fieldName", "value": "''"}], "start": 1678768835978, "stop": 1678768836060}, {"name": "Взять текст из элемента ''Текст ошибки поля  пароль не верный или текст отсутствует''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000001B11621E940>"}, {"name": "allureText", "value": "'Текст ошибки поля  пароль не верный или текст отсутствует'"}], "start": 1678768843105, "stop": 1678768843137}, {"name": "Заполнение поля '' текстом ''mail@mail.r''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000001B11621C640>"}, {"name": "text", "value": "'mail@mail.r'"}, {"name": "fieldName", "value": "''"}], "start": 1678768845140, "stop": 1678768845230}, {"name": "Взять текст из элемента ''Текст ошибки поля  пароль не верный или текст отсутствует''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000001B11621E940>"}, {"name": "allureText", "value": "'Текст ошибки поля  пароль не верный или текст отсутствует'"}], "start": 1678768852281, "stop": 1678768852329}, {"name": "Заполнение поля '' текстом ''mail@''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000001B11621C640>"}, {"name": "text", "value": "'mail@'"}, {"name": "fieldName", "value": "''"}], "start": 1678768854329, "stop": 1678768854454}, {"name": "Взять текст из элемента ''Текст ошибки поля  пароль не верный или текст отсутствует''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000001B11621E940>"}, {"name": "allureText", "value": "'Текст ошибки поля  пароль не верный или текст отсутствует'"}], "start": 1678768861498, "stop": 1678768861515}, {"name": "Заполнение поля '' текстом ''мейл@mail.ru''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000001B11621C640>"}, {"name": "text", "value": "'мейл@mail.ru'"}, {"name": "fieldName", "value": "''"}], "start": 1678768863517, "stop": 1678768863609}, {"name": "Взять текст из элемента ''Текст ошибки поля  пароль не верный или текст отсутствует''", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'некорректный e-mail')]\")).text\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный e-mail')]\"}\n  (Session info: chrome=111.0.5563.65)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678768814622\\1678768814623.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678768814622\\1678768814623.html\n\n", "trace": "  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\base.py\", line 16, in get_element_text\n    return element.get(query.text)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 116, in get\n    return self.wait.for_(query)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\wait.py\", line 125, in for_\n    raise self._hook_failure(failure)\n"}, "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x000001B11621E940>"}, {"name": "allureText", "value": "'Текст ошибки поля  пароль не верный или текст отсутствует'"}], "start": 1678768870681, "stop": 1678768880829}], "start": 1678768817645, "stop": 1678768880831}], "attachments": [{"name": "Скринтшот", "source": "734b748f-e24a-455b-8ac3-f41eeaa6969e-attachment.png", "type": "image/png"}, {"name": "Логи консоли", "source": "44efab39-5603-43bb-983c-fb5b3f480637-attachment.json", "type": "application/json"}], "start": 1678768817417, "stop": 1678768880831, "uuid": "47eddd5d-44ac-4bf8-8380-6c7491888d14", "historyId": "0cc81e0474bc006509c2841748279852", "testCaseId": "2a3cb9a75e79c243aed59c4006152af4", "fullName": "tests.test_login.TestLogin#test_field_email", "labels": [{"name": "feature", "value": "Тесты логина"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-VLT8PU4"}, {"name": "thread", "value": "8628-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}